src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -59,33 +59,28 @@
  */
 public final class InstantiatingChannelProvider implements ChannelProvider {
   private static final String DEFAULT_GAX_VERSION = "UNKNOWN";
+  private static final String DEFAULT_GRPC_VERSION = "1.0.1";
```

Comment:
Is this going to be used in practice? It seems like we're going to forget about it and allow it to fall out of date, which taints our metrics. I'd rather report UNKNOWN or nothing at all, if there are corner cases in which we cannot detect this.